Library/Homebrew/utils/tty.rb

```diff
@@ -6,7 +6,7 @@ def strip_ansi(string)
   end
 
   def width
-    `/usr/bin/tput cols`.strip.to_i
+    `/bin/stty size` =~ /(\d+)$/ ? $1.to_i : 80
```

Comment:
Personally I don't know when stty would fail, it should always succeed, but better safe then sorry.